lib/implore.es6.js

```diff
@@ -121,7 +121,16 @@ export default function implore(request, callback) {
 			response.body = xhr.responseText;
 			response.status = xhr.status;
 			response.type = xhr.getResponseHeader('Content-Type');
-			response.headers = xhr.getAllResponseHeaders();
+			const rawHeaders = xhr.getAllResponseHeaders();
+			// Just in case some headers don't get processed properly
+			response.rawHeaders = rawHeaders;
+			// Processed headers as a hash for easier use
+			const responseHeaders = rawHeaders.split("\n");
+			response.headers = {};
+			for (let header of responseHeaders) {
+				const parts = header.split(': ');
```

Comment:
While it's standard (and done almost everywhere of which I'm aware), I don't think that a space after the colon is required; I think it's optional. (https://tools.ietf.org/html/rfc7230#section-3.2.3) That being the case, could you simply split on the colon alone and trim?